src/common/utils/registry.h

```diff
@@ -51,11 +74,28 @@ namespace registry
         {
         }
 
+        std::wstring toString() const
+        {
+            using namespace detail;
+
+            std::wstring value_str;
+            std::visit(overloaded{ [&](DWORD value) {
+                                      std::wostringstream oss;
+                                      oss << value;
+                                      value_str = oss.str();
+                                  },
+                                   [&](const std::wstring& value) { value_str = value; } },
+                       value);
+
+            return fmt::format(L"{}\\{}\\{}:{}", detail::getScopeName(scope), path, name ? *name : L"Default", value_str);
+        }
+
         bool isApplied() const
         {
             HKEY key{};
-            if (RegOpenKeyExW(scope, path.c_str(), 0, KEY_READ, &key) != ERROR_SUCCESS)
+            if (auto res = RegOpenKeyExW(scope, path.c_str(), 0, KEY_READ, &key); res != ERROR_SUCCESS)
             {
+                Logger::info(L"isApplied of {}: RegOpenKeyExW failed: {}", toString(), get_last_error_or_default(res));
```

Comment:
should this be ::error?

Reply:
`isApplied` is a method for checking whether a registry entry is there, so failing to open/query it is not an error.